docker-compose.yml

```diff
@@ -63,10 +63,14 @@ services:
       context: ./infrastructure/containers/postgres/automatic-backup/
       dockerfile: Dockerfile
     environment:
+      BACKUP_BUCKET: rootski-database-backups
       BACKUP_DIR: /backups
       BACKUP_INTERVAL: 1h
```

Comment:
You know, I know this is a surprise ask, but most contributors won't have access to the database backup bucket. If they did, they'd have access not only to our whole dataset, but also to PII data if we start to collect that from customers. (We already collect emails, so I guess we do this).

This means that this container won't work for them because it works with S3. Could we make it so that if their credentials don't have read/write access, the backup/restore features just act like they used to? (reading/writing to local disk). 

We could move that to a separate PR/task if that's not something you want to do right now. The way we could implement it is:

On startup, try to read something trivial (maybe just listing the objects in the bucket) from the s3 bucket. If that fails, read/write to local disk, if it succeeds, upload/download from the s3 bucket as you're doing here.

Reply:
BTW, if we added this logic, I think we'd call a function like `aws_credentials_have_read_access_to_db_backups_bucket(iam_secret_key_id: str, iam_secret_key: str) -> bool:`

From there, values could be passed as parameters to the backup/restore functions that make sure the correct behavior is used.

Reply:
I'll do this as a separate PR but this is an excellent point